### DISCUSSION TITLE
Notification Webhook

### DISCUSSION THREAD

**Discussion by marcuuscardoso** [mercadopago/sdk-js#77] [Q&A] [answered]

**Comment by lucmantovani** [comment, marked as answer]
Olá @Marcus-Cardoso!
Tudo bem?

Esse espaço é dedicado para discussões e relatos relacionados às SDK em si.
Para dúvidas de integração, o canal recomendado é a [Central de atendimento para desenvolvedores](https://www.mercadopago.com.br/developers/pt/support/center/tickets).
Lá temos pessoas com conhecimento de todo o processo de integração, inclusive relacionado à notificações!
